src/View/TextPanel.java

```diff
@@ -1,158 +0,0 @@
-package View;
```

Comment:
Is there a reason why this is completely being deleted and then re-added later in the changes list?